docs/docs/guides/social/github.mdx

```diff
@@ -8,124 +8,203 @@ sidebar_custom_props:
   docCardIconName: github-dark
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # Login with GitHub
 
 To enable GitHub login for your project, you need to
-- Create a GitHub OAuth application
-- Create and add your Google OAuth credentials to your Hanko backend configuration
-- Add the Hanko `auth` web component or add the required code to your JS Client App using the `hanko-frontend-sdk`.
+- Create a GitHub OAuth application.
+- Create your GitHub OAuth credentials and configure them with Hanko.
+- Add the Hanko `<hanko-auth>` web component or add the required code to your frontend using the `hanko-frontend-sdk`.
 
 ## Access your GitHub account
 
 - Go to [github.com](https://github.com).
-- Click on 'Sign In' at the top right to log in. If you do not have an account yet, create one.
+- Click on `Sign In` at the top right to log in. If you do not have an account yet, create one.
+
+## Create a GitHub OAuth App {#create-app}
```

Comment:
Should we distinguish here between Personal and Organizational OAuth Applications? The flow to do this is slightly different in that case.
Top Right. Your Organizations -> Settings -> Developer Settings -> Oauth Apps.